lib/rex/post/postgresql/ui/console.rb

```diff
@@ -40,6 +43,7 @@ def initialize(session)
 
             enstack_dispatcher(::Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher::Core)
             enstack_dispatcher(::Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher::Client)
+            enstack_dispatcher(::Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher::LocalFs)
```

Comment:
I think we could just have a SQL single dispatcher implemented, and we could update this to be:

```diff
diff --git a/lib/rex/post/postgresql/ui/console.rb b/lib/rex/post/postgresql/ui/console.rb
index 0f2d50d4ea..50938a7e52 100644
--- a/lib/rex/post/postgresql/ui/console.rb
+++ b/lib/rex/post/postgresql/ui/console.rb
@@ -15,13 +15,12 @@ module Rex
         class Console
           include Rex::Post::Sql::Ui::Console
           include Rex::Ui::Text::DispatcherShell
-          include Rex::Post::LocalFs
 
           # Dispatchers
           require 'rex/post/postgresql/ui/console/command_dispatcher'
           require 'rex/post/postgresql/ui/console/command_dispatcher/core'
           require 'rex/post/postgresql/ui/console/command_dispatcher/client'
-          require 'rex/post/postgresql/ui/console/command_dispatcher/local_fs'
+          require 'rex/post/sql/ui/console/command_dispatcher/local_fs'
 
           #
           # Initialize the PostgreSQL console.
@@ -43,7 +42,7 @@ module Rex
 
             enstack_dispatcher(::Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher::Core)
             enstack_dispatcher(::Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher::Client)
-            enstack_dispatcher(::Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher::LocalFs)
+            enstack_dispatcher(::Rex::Post::Sql::Ui::Console::CommandDispatcher::LocalFs)
 
             # Set up logging to whatever logsink 'core' is using
             if ! $dispatcher['postgresql']
diff --git a/lib/rex/post/postgresql/ui/console/command_dispatcher/local_fs.rb b/lib/rex/post/postgresql/ui/console/command_dispatcher/local_fs.rb
deleted file mode 100644
index d56edbd469..0000000000
--- a/lib/rex/post/postgresql/ui/console/command_dispatcher/local_fs.rb
+++ /dev/null
@@ -1,21 +0,0 @@
-# -*- coding: binary -*-
-
-require 'rex/post/sql/ui/console/command_dispatcher/local_fs'
-
-module Rex
-  module Post
-    module PostgreSQL
-      module Ui
-        ###
-        #
-        # PostgreSQL local file system commands
-        #
-        ###
-        class Console::CommandDispatcher::LocalFs
-          include Rex::Post::Sql::Ui::Console::CommandDispatcher::LocalFs
-          include Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher
-        end
-      end
-    end
-  end
-end
diff --git a/lib/rex/post/sql/ui/console/command_dispatcher/local_fs.rb b/lib/rex/post/sql/ui/console/command_dispatcher/local_fs.rb
index 44bf1f6d43..fbc53dab62 100644
--- a/lib/rex/post/sql/ui/console/command_dispatcher/local_fs.rb
+++ b/lib/rex/post/sql/ui/console/command_dispatcher/local_fs.rb
@@ -11,7 +11,7 @@ module Rex
             #
             # SQL File system commands
             #
-            module LocalFs
+            class LocalFs
               include Rex::Post::Sql::Ui::Console::CommandDispatcher
               include Rex::Post::LocalFs
 
```

And obviously we'd need to do the same for mssql etc

Not a blocker: it'd be great if there was just a single implementation, that would be used for both smb and SQL